### DISCUSSION TITLE
icon availability inconsistency b/w versions

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1354] [Q&A]
the `admin` icon which rendered correctly in versions 0.18.0, 0.19.0, and 0.20.0 -- seems to be unavailable in v0.21.0. Not sure whether that's a bug or on purpose. The icon is however available in fluent ui icons list.

**Comment by mturoci** [comment]
That's expected as we switched the icon set. Seems like Fluent might removed/renamed some icons.

>The icon is however available in fluent ui icons list.

The correct list to look at is https://developer.microsoft.com/en-us/fluentui#/styles/web/icons => Available icons => SVG icons or even simpler - our [docs](https://wave.h2o.ai/docs/icons)
